gbfs-validator/versions/partials/v3.0-RC2/system_information/required_store_uri.js

```diff
@@ -0,0 +1,61 @@
+module.exports = ({ android = false, ios = false }) => {
+  const r = {
```

Comment:
What does "r" mean?

Reply:
Thank you for the suggestion! I renamed the variable to `partial`.